<launch>
  <node pkg="rosbag" type="play" name="rosbag" args="-d 2 /home/nathan/records/record4.bag"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/head_xtion/rgb/image_color"/>
  </node>
</launch>
